internal/ingress/controller/config/config.go

```diff
@@ -712,6 +715,7 @@ func NewDefault() Configuration {
 		EnableUnderscoresInHeaders:       false,
 		ErrorLogLevel:                    errorLevel,
 		UseForwardedHeaders:              false,
+		EnableRealIp:                     false,
```

Comment:
Not sure if this is needed, @ElvinEfendi can you confirm?

Reply:
Yes, we want to default this to `false`. (It probably defaults to `false` when unset but better be explicit)